packages/firestore/src/util/log.ts

```diff
@@ -19,37 +19,42 @@
 import { SDK_VERSION } from '../core/version';
 import { AnyJs } from './misc';
 import { PlatformSupport } from '../platform/platform';
+import { Logger, LogLevel as _LogLevel } from '@firebase/logger';
```

Comment:
_LogLevel => FirebaseLogLevel ?  (maybe Logger => FirebaseLogger too for symmetry)

Reply:
SGTM